vendor/github.com/vmware/go-nfs-client/nfs/rpc/client.go

```diff
@@ -25,6 +26,9 @@ const (
 	Success = iota
```

Comment:
perhaps somewhere around these const definitions we might make a const for `"tcp"`